source/documentation/syntax/comments.liquid

```diff
@@ -18,27 +22,27 @@ compiled to a CSS comment. They're also called **loud comment**, by contrast
 with silent comments. A multi-line comment that's compiled to CSS may contain
 [interpolation][], which will be evaluated before the comment is compiled.
 
-By default, multi-line comments be stripped from the compiled CSS in [compressed
-mode][]. If a comment begins with `/*!`, though, it will always be included in
-the CSS output.
+By default, multi-line comments will be stripped from the compiled CSS in
+[compressed mode][]. If a comment begins with `/*!`, though, it will always be
+included in the CSS output.
 
 [statement]: /documentation/syntax/structure#statements
 [interpolation]: /documentation/interpolation
 [compressed mode]: /documentation/cli/dart-sass#style
 {% endmarkdown %}
 
-{% codeExample 'scss-comment', , true, 'scss'%}
```

Comment:
@sanajaved7 Was there a reason to use this extra comma? I couldn't see it having any effect.

Reply:
@jgerigmeyer I saw that too and think that might have been in there by mistake!